Gemfile

```diff
@@ -14,6 +14,10 @@ gem 'jbuilder', '~> 2.5'
 gem 'mysql2', '>= 0.3.18', '< 0.5'
 gem 'haml-rails'
 gem 'bootstrap-sass'
+gem 'faker'
```

Comment:
this gem needs only for test env? If true, we need move it under corresponding env